src/manifold/src/quickhull.cpp

```diff
@@ -78,13 +78,15 @@ size_t MeshBuilder::addHalfedge() {
     return index;
   }
   halfedges.push_back({});
+  halfedgeToFace.push_back(0);
```

Comment:
Face index is needed in `quickhull`, so I keep a `halfedgeToFace` vector for that.